fe/fe-core/src/main/java/org/apache/doris/common/proc/PartitionsProcDir.java

```diff
@@ -380,6 +380,9 @@ private List<Pair<List<Comparable>, TRow>> getPartitionInfosInrernal() throws An
                     trow.addToColumnValue(new TCell().setStringVal(FeConstants.null_string));
                 }
 
+                partitionInfo.add(partition.getCommittedVersion());
```

Comment:
What's the result of CloudPartition.getCommittedVersion()?

Reply:
cloud partition no committed version,  no real meaning,   no changed